scripts/ngless-cwl/cwl/ngless-count.cwl

```diff
@@ -6,49 +6,48 @@
 cwlVersion: "cwl:v1.0"
 
 class: CommandLineTool
-baseCommand: ['./ngless-count.py']
+baseCommand: ['ngless-count.py']
+
+requirements:
+- $import: ngl-types.yml
 
 doc: |
   None
 
 inputs:
-  
   input:
-    type: str
-  
+    type: File
     doc: SAM/BAM/CRAM file to count reads on
     inputBinding:
-      prefix: --input 
+      prefix: --input
 
   output:
-    type: str
-  
+    type: string
     doc: Output file/path for results
```

Comment:
You can simplify usage of these CWL descriptions by either
1) providing a default output file name 
or
2) removing the option and specify a fixed name in `arguments`